### PR TITLE
[new] Add qartjs@1.0.2 w/ npm auto-update

### PR DESCRIPTION
**Pull request by ironprice91**
Adding qart.js by using the cdnjs-importer script
@PeterDaveHello @maruilian11 do the updates here look ok (specifically the qartjs/package.json)? 
Please let me know if I need to update anything, thanks :) 

Pull request for issue: #10499
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kciter/qart.js
 * Official website (optional, not the repository): https://kciter.github.io/qart.js/
 * NPM package url (optional): https://npmjs.com/package/qartjs
 * License and its reference: [GPLv3](https://github.com/kciter/qart.js/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 53
   - Count of stars: 2,071
   - Count of forks: 82
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 20
   - Downloads in the last month: 175

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/qartjs/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "filename": "qart.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kciter/qart.js.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "qart.js",
+          "qart.min.js"
```
Comment (pvnr0082t) [inline ajax/libs/qartjs/package.json]: Let's use `**/*` instead of these two rules. Thanks.

File: ajax/libs/qartjs/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "author": "Lee Sun-Hyoup <kciter@naver.com>",
+  "license": "GPL-3.0",
+  "filename": "qart.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kciter/qart.js.git",
+    "fileMap": [
+      {
+        "basePath": "",
```
Comment (pvnr0082t) [inline ajax/libs/qartjs/package.json]: Let's change the basePath to `"dist"`.

File: ajax/libs/qartjs/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "qrcode"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kciter/qart.js.git"
+  },
+  "author": "Lee Sun-Hyoup <kciter@naver.com>",
+  "license": "GPL-3.0",
+  "filename": "qart.min.js",
+  "autoupdate": {
```
Comment (pvnr0082t) [inline ajax/libs/qartjs/package.json]: There are 3 releases on NPM and the latest version is newer than GitHub. Let's transfer to use npm auto-update config. Please take a look at [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).

**Comment by ironprice91** [comment]
@pvnr0082t Updated the package.json to use the npm auto-update config, added homepage info + updated fileMap info.  Please let me know if there anymore changes, thanks. :)

**Comment by maruilian11** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@ironprice91 would you please clean up the commit message to `Add qartjs@1.0.2 w/ npm auto-update`? Thanks.

File: ajax/libs/qartjs/package.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "qart.min.js",
+  "autoupdate": {
```
Comment (maruilian11) [inline ajax/libs/qartjs/package.json]: we only need one auto-update, why also add git auto-update config ?

Comment (ironprice91) [inline ajax/libs/qartjs/package.json]: @maruilian11 Sorry about that - I think rebased incorrectly. Will update in a few, thanks.

Comment (maruilian11) [inline ajax/libs/qartjs/package.json]: thank you

**Comment by ironprice91** [comment]
@maruilian11  @pvnr0082t I think I'm having some issues updating my local repo with upstream master. 

I did the following steps:

git pull rebase
git rebase upstream/master

From here it would get stuck at
"First, rewinding head to replay your work on top of it..."

Any advice for me to fix?

**Comment by ironprice91** [comment]
@PeterDaveHello  @maruilian11 @pvnr0082t - Sorry about that guys. I re-added the commits in there and re-ran the tests. Re-opening the PR, could you take a look? Thanks

**Comment by PeterDaveHello** [comment]
The commit message duplicated its self.

**Comment by PeterDaveHello** [comment]
I just updated the commit message, @ironprice91 but your email in the commit seems to be not in your GitHub account, if you'll like this contribution be counted in your account, you'll need to add the email into your account.

**Comment by PeterDaveHello** [comment]
@pvnr0082t @maruilian11 please don't forget to check the commit message ...

**Pull request merged**